src/default-config.ls

```diff
@@ -0,0 +1,53 @@
+require! <[ path fs ]>
+
+{ filter, map, first, join, keys } = require 'prelude-ls'
+
+# RC automatically overwrites these with env variables.
+# For example to edit environment set arch_environment
+
+/* Map of parsers which take a file path and parse functions*/
+
+parsers =
+  js: -> require it,
+  ls: -> require it
+
+parser = (fname) -> parsers[(path.extname fname).slice(1)](fname)
+
+fpath-regex = new RegExp "arch\.config\.(?:#{parsers |> keys |> join '|'})$"
+
+filter-configs = -> fpath-regex.test it
+
+merge = (x, xs) -> x import xs
+
+initial-conf =
+  app-path:     process.env.arch_app_path or path.resolve '.'
```

Comment:
Think of it this way: `express` also doesn't change anything based on environment, because it's easy to do it yourself. Now we have a config file, which makes it easy to do yourself, so we don't need to add more code and convention to maintain.